include/bluetooth/audio/vcs.h

```diff
@@ -0,0 +1,504 @@
+
+/**
+ * @brief Unmute the server.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_unmute(struct bt_conn *conn);
+
+/**
+ * @brief Mute the server.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_mute(struct bt_conn *conn);
+
+/**
+ * @brief Read the Volume Offset Control Service offset state.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Volume Offset Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_state_get(struct bt_conn *conn, struct bt_vocs *inst);
+
+/**
+ * @brief Read the Volume Offset Control Service location.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Volume Offset Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_location_get(struct bt_conn *conn, struct bt_vocs *inst);
+
+/**
+ * @brief Set the Volume Offset Control Service location.
+ *
+ * @param conn       Connection to peer device, or NULL to read local server
+ *                   value.
+ * @param inst       Pointer to the Volume Offset Control Service instance.
+ * @param location   The location to set.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_location_set(struct bt_conn *conn, struct bt_vocs *inst,
+			     uint8_t location);
+
+/**
+ * @brief Set the Volume Offset Control Service offset state.
+ *
+ * @param conn    Connection to peer device, or NULL to read local server value.
+ * @param inst    Pointer to the Volume Offset Control Service instance.
+ * @param offset  The offset to set (-255 to 255).
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_state_set(struct bt_conn *conn, struct bt_vocs *inst,
+			  int16_t offset);
+
+/**
+ * @brief Read the Volume Offset Control Service output description.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Volume Offset Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_description_get(struct bt_conn *conn, struct bt_vocs *inst);
+
+/**
+ * @brief Set the Volume Offset Control Service description.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server
+ *                      value.
+ * @param inst          Pointer to the Volume Offset Control Service instance.
+ * @param description   The description to set. Value will be copied.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_description_set(struct bt_conn *conn, struct bt_vocs *inst,
+				const char *description);
+
+/**
+ * @brief Deactivates a Audio Input Control Service instance.
```

Comment:
"Deactivates" -> "Deactivate", to be consistent with the imperative form of the other functions.

"a" -> "an"